docs/rpi3.md

```diff
@@ -206,44 +206,42 @@ $ sudo rm -rf /srv/nfs/rpi/boot/*
 ```
 
 ### 5.4 Update uboot.env
-We need to make a couple of changes to that file to ensure that it will try to
-boot using everything we have prepared. So, start by inserting the UART cable
-and open up `/dev/ttyUSB0`
-```bash
+Actually, there are two ways to update uboot.env. First, you can edit
+`build/rpi3/firmware/uboot.env.txt` file, which is used as simple text source for
+generation of uboot.env during build. And you can just edit u-boot env via UART
```

Comment:
Change from:
` uboot.env during build. And you ...`
to 
` uboot.env during build or you ...`